ui/src/shared/utils/useVisDomainSettings.ts

```diff
@@ -22,7 +23,7 @@ export const getValidRange = (
   timeRange: TimeRange | null
 ) => {
   const range = extent((data as number[]) || [])
-  if (!timeRange) {
+  if (isNull(timeRange)) {
```

Comment:
this never sent up with my last PR so I'm including here as per the PR suggestions